homebrew/using_different_homebrew_formula_versions.md

```diff
@@ -0,0 +1,35 @@
+# Using different Homebrew formula versions
+
+Today, I was trying to run the executable from a gem and got the following error:
+
+```bash
+Users/jessieayoung/.rbenv/versions/2.1.3/lib/ruby/gems/2.1.0/gems/rmagick-2.13.2/lib/RMagick.rb:11:in
+`require': This installation of RMagick was configured with ImageMagick 6.8.9
+but ImageMagick 6.9.0-3 is in use. (RuntimeError)
+```
+
+My first instinct was to uninstall ImageMagick 6.9.0 and then install 6.8.9 with
+Homebrew. Through The Google, I learned that Homebrew saves older formula
+versions. You can see which versions you have like this (notice the asterisk
+next to the version in use):
+
+```bash
+% brew info imagemagick
+imagemagick: stable 6.9.0-3 (bottled), HEAD
+http://www.imagemagick.org
+/usr/local/Cellar/imagemagick/6.8.8-9 (1429 files, 21M)
+  Poured from bottle
+/usr/local/Cellar/imagemagick/6.8.9-1 (1432 files, 22M)
+  Poured from bottle
+/usr/local/Cellar/imagemagick/6.9.0-3 (1440 files, 22M) *
+  Poured from bottle
+```
```

Comment:
I like it! Always a tricky balance between providing accurate and readable output, but I think the above is both :)